src/pks/transport/transport_ats_pk.cc

```diff
@@ -52,6 +53,17 @@ Transport_ATS::Transport_ATS(Teuchos::ParameterList& pk_tree,
   PK(pk_tree, glist, S, solution),
   PK_PhysicalExplicit<Epetra_Vector>(pk_tree, glist, S, solution)
 {
+
+
+  key_ = Keys::readKey(*plist_, domain_, "primary variable", "total_component_concentration");
+  tcc_key_ = key_;
+  
+  // set up the primary variable solution, and its evaluator
+  Teuchos::ParameterList& FElist = S->FEList();
```

Comment:
Oh wow, was TCC not already a primary variable evaluator?  I guess not.  Glad this is happening!